Index: target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst b/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst
--- a/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst	
+++ b/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst	
@@ -1,0 +1,1 @@
+/home/sajal/IdeaProjects/untitled/sajaal_maven/miniproject/src/test/java/CalculatorTest.java
